src/makielayout/blocks/polaraxis.jl

```diff
@@ -0,0 +1,537 @@
+# First, define the polar-to-cartesian transformation as a Makie transformation
+# which is fully compliant with the interface
+
+struct PolarAxisTransformation
+    θ_0::Float64
+    direction::Int
+end
```

Comment:
This should probably have a docstring or a comment on what θ_0 and direction are